Dockerfile

```diff
@@ -0,0 +1,81 @@
+# This Dockerfile contains multiple targets.
+# Use 'docker build --target=<name> .' to build one.
+#
+# Every target has a BIN_NAME argument that must be provided via --build-arg=BIN_NAME=<name>
+# when building.
+
+# ===================================
+#
+#   Release images.
+#
+# ===================================
+
+
+# default release image
+# -----------------------------------
+FROM docker.mirror.hashicorp.services/alpine:latest AS release-default
```

Comment:
CRT wants the Dockerfile at the root of the repo, and their examples encourage this style of using multi-stage builds.

Basically, this lets us stick config for multiple images in one Dockerfile. For now, I just have one image ("release-default") to get us started, so all the CRT workflows can run, but this will likely change as I finish things up.